docs/spatial-raster.rst

```diff
@@ -49,3 +49,57 @@ Example:
 
     # Stack landsat tif files
     arr, arr_meta = es.stack_raster_tifs(all__paths, destfile)
+
+
+Draw Legend
```

Comment:
Very nice submission @hdumke !! One small change. While this does render just fine, normally it is requested that rst headings are the same line length. So I am requesting you remove a few of the tildas `~` like i did below. Once you do that I will approve and merge your pr! 

```
Draw Legend
~~~~~~~~~~~
```

Reply:
@lwasser made the requested change to remove the extraneous tildas so that they only underline the heading.